episodes/02-lesson-design.md

```diff
@@ -36,8 +36,13 @@ to directly meet those stated outcomes.
 4. Assess learner progress towards desired outcomes
 5. Evaluate chosen outcomes, experiences, and content based on this assessment
 
-**TODO: add a figure illustrating the setps in the list above.
-could be based on Fig. 2 in [f1000-course-design-guide]**
+![Nicholl's five phases of curriculum design, 
+presented as a cycle to reflect that all aspects of the design should be 
+revisited in response to the results of the evaluation that takes place at step 5.
+](fig/nicholls-five-phases.svg){
+alt="A flow diagram presenting the Nicholl's five phases of curriculum design as a cycle."
+width="67%"
+}
```

Comment:
Maybe a `[^1]` reference could be good here to point to the original paper?

Reply:
I will address this in the branch for #208